setup.py

```diff
@@ -67,3 +67,48 @@
     compression=None,
     row_group_size=BATCH_SIZE,
 )
+
+# %%
+# save to tensorstore
+sharded_dense_chunk = ts.open(
```

Comment:
Unless I misunderstand: could we call this `chunked` here instead of `sharded`?

When we talk about sharding, we typically mean sharding tables (or arrays) at a "macro scale", meaning 100k or millions of rows per shard.

"Chunking" is what's relevant for batch-loading and operates on the "micro scale", the batch size, something between 32 and 2048 or so.

Reply:
["Sharding is the name used in the zarr world"](https://zarr.dev/zeps/accepted/ZEP0002.html) for saving multiple chunks within a single file object (which is what is being done here and in theory should lead to performance improvements due to increased parallelization).  See also https://google.github.io/tensorstore/driver/zarr3/index.html#json-driver/zarr3/Codec/sharding_indexed

Reply:
I see - it's a fair naming choice given how similar it is; but parquet and databases are so much older and have been using the term "sharding" on the "macro scale".

Do you mind me calling this "chunked" in the context of all other methods and making a note that it's called "shared" in the zarr world?

Reply:
Go for it!